Directory.Packages.props

```diff
@@ -0,0 +1,46 @@
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Azure.Identity" Version="1.11.2" />
+    <PackageVersion Include="Azure.Messaging.ServiceBus" Version="7.17.5" />
+    <PackageVersion Include="Fluid.Core" Version="2.8.0" />
+    <PackageVersion Include="Jint" Version="3.0.0" />
+    <PackageVersion Include="JsonSchema.Net" Version="3.3.2" />
+    <PackageVersion Include="McMaster.Extensions.CommandLineUtils" Version="3.0.0" />
+    <PackageVersion Include="MessagePack" Version="2.2.85" />
+    <PackageVersion Include="Microsoft.AspNetCore.Hosting.WindowsServices" Version="5.0.12" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.8" />
+    <PackageVersion Include="Microsoft.Azure.Relay" Version="3.0.1" />
+    <PackageVersion Include="Microsoft.Azure.Relay.AspNetCore" Version="1.3.15173" />
+    <PackageVersion Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.2.0" />
+    <PackageVersion Include="Microsoft.Diagnostics.NETCore.Client" Version="0.2.328102" />
+    <PackageVersion Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="3.0.5" />
+    <PackageVersion Include="Microsoft.Extensions.Caching.Memory" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration.CommandLine" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.FileProviders.Abstractions" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="all" />
+    <PackageVersion Include="Microsoft.Windows.CsWin32" Version="0.3.49-beta" PrivateAssets="all" />
+    <PackageVersion Include="Mono.Cecil" Version="0.11.3" />
+    <PackageVersion Include="Mono.Unix" Version="7.1.0-final.1.21458.1" />
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.3"/>
+    <PackageVersion Include="NuGet.Versioning" Version="5.6.0" />
+    <PackageVersion Include="Octokit" Version="10.0.0" />
+    <PackageVersion Include="Parlot" Version="0.0.25" />
+    <PackageVersion Include="Serilog.AspNetCore" Version="4.1.0" />
+    <PackageVersion Include="Serilog.Formatting.Compact" Version="1.1.0" />
+    <PackageVersion Include="Serilog.Sinks.Console" Version="4.0.0" />
+    <PackageVersion Include="Serilog.Sinks.File" Version="5.0.0" />
+    <PackageVersion Include="System.CommandLine" Version="2.0.0-beta1.20427.1" />
+    <PackageVersion Include="System.IO.Hashing" Version="8.0.0" />
+    <PackageVersion Include="System.IO.Pipelines" Version="4.6.0" />
+    <PackageVersion Include="System.Linq.Async" Version="4.1.1" />
+    <PackageVersion Include="System.Text.Encodings.Web" Version="4.7.2" AllowExplicitReference="true" />
```

Comment:
Interesting, first time I see the AllowExplicitReference. Do you know what is that for?

Reply:
https://github.com/dotnet/aspnetcore/blob/main/eng/targets/ResolveReferences.targets#L68

Talking with Brennan it could be related to the fact that it's part of the shared framework, and that it could complain otherwise?